pkgs/development/python-modules/univers/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, fetchPypi
+, fetchpatch
+, buildPythonPackage
+, setuptools-scm
+, attrs
+, packaging
+, pyparsing
+, semantic-version
+, semver
+, black
+, commoncode
+, pytestCheckHook
+, saneyaml
+}:
+
+buildPythonPackage rec {
+  pname = "univers";
+  version = "30.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-yM0SDBpkiZEbaZ0ugjiMwwUFKqZGbmh1JNlv5qvPAYo=";
+  };
+
+  patches = [
+    # Make tests work when not using virtualenv, can be dropped with the next version
+    # Upstream PR (already merged): https://github.com/nexB/univers/pull/77
+    (fetchpatch {
+      url = "https://github.com/nexB/univers/commit/b74229cc1c8790287633cd7220d6b2e97c508302.patch";
+      sha256 = "sha256-i6zWv9rAlwCMghd9g5FP6WIQLLDLYvp+6qJ1E7nfTSY=";
+    })
+  ];
+
+  nativeBuildInputs = [ setuptools-scm ];
+  propagatedBuildInputs = [ attrs packaging pyparsing semantic-version semver ];
+  checkInputs = [ black commoncode pytestCheckHook saneyaml ];
```

```suggestion
  checkInputs = [ commoncode pytestCheckHook saneyaml ];
```
we don't want to run linting tests.

---

black is a code formatter. not needed to evaluate the tests

---

Then please disable/ignore that test